### ISSUE TITLE
Parser changed unexpected after rio merge #954

### ISSUE THREAD

**Issue by Michiel-s** [AmpersandTarski/Ampersand#960]
I think due to #954 the following syntax is not valid anymore. Please fix asap. This really is a blocking issue for me.

ADL syntax that should work:
```adl
VIEW "Note"   : Note HTML TEMPLATE "Note.html" ENDVIEW
```
But this results in error: 
```
unexpected keyword "HTML"
  expecting keyword "DEFAULT", symbol '{' or symbol '('
```

**Comment by RieksJ** [comment]
@Michiel-s please add the ampersand-version that you are using.
Does the following syntax work (I added `{` and `}`?
~~~
VIEW "Note": Note { } HTML TEMPLATE "Note.html" ENDVIEW
~~~

**Comment by hanjoosten** [comment]
Could you supply a (minimal?) test script?

**Comment by hanjoosten** [comment]
I found the cause. I changed some lists to be non-empty. I was mistakenly thought that all views should have at least one attribute in its list. There is no test for this in the regression set at Travsi, otherwise I would not have made this change.
Obviously this was wrong. I'll fix it asap.

**Comment by Michiel-s** [comment]
Fixed by #961
